### DISCUSSION TITLE
How to disable Markdown rendering effects

### DISCUSSION THREAD

**Discussion by ilveroluca** [LazyVim/LazyVim#2729] [Q&A] [answered]
Hi there,

I'm new to LazyVim and starting to feel my way around. When editing Markdown files, the default configuration applies effects to how the text is rendered.  I'd like to turn them all off.

I have already read about setting [conceallevel](https://neovim.io/doc/user/options.html#'conceallevel') to 0.  I did that, and it got rid of some stuff.  However, headings and whitespace are still rendered with some formatting and concealing the actual characters (see screenshots below, the first with file type `markdown` and the second with file type `text`).

Any idea about how I can totally turn off these rendering effects?

![rendering as markdown](https://github.com/LazyVim/LazyVim/assets/1502936/a643b02d-ed30-4b1b-ba97-d8c71d977d77)

![rendering as text](https://github.com/LazyVim/LazyVim/assets/1502936/0de0e704-4e54-4830-8ece-bab4dbb9648a)

**Comment by dpetka2001** [comment, marked as answer]
Disable plugin `headlines.nvim`.

**Reply by ilveroluca** [reply]
Thank you!
